src/source/parser.scala

```diff
@@ -33,6 +33,7 @@ case class Parser() {
 
 val visitedFiles = mutable.Set[File]()
 val fileStack = mutable.Stack[File]()
+def fileParent:String = if (fileStack.top.getParent() != null) return fileStack.top.getParent() + "/" else return ""
```

Comment:
Can this become an inner declaration inside of importFile?
I worry that its behavior is tailored for a specific use case (given the use of filestack.top), so it doesn't really belong as a general method of this class.  But as a local helper it's great.